core/src/test/java/org/apache/spark/JavaAPISuite.java

```diff
@@ -733,8 +733,10 @@ public Boolean call(Double x) {
     assertEquals(20/6.0, rdd.mean(), 0.01);
     assertEquals(20/6.0, rdd.mean(), 0.01);
     assertEquals(6.22222, rdd.variance(), 0.01);
+    assertEquals(rdd.variance(), rdd.popVariance(), 0.01);
```

Comment:
I think these need to assert exact equality; there's no reason that they would ever be different at all.